### PR TITLE
Rise from PEEP to PIP in 100 ms in pressure assist mode

### PR DESCRIPTION
**Pull request by asmodai27**
Reuse of the code from pressure control mode.
I assumed that RISE_TIME is not dependent on the ventilator mode, please correct me if I am wrong.

### CODE REVIEW

**Comment by jlebar** [comment]
> I assumed that RISE_TIME is not dependent on the ventilator mode

That's my assumption as well.

File: controller/lib/core/blower_fsm.h [hunk 1]
```diff
@@ -84,6 +84,10 @@ struct BlowerSystemState {
   bool is_new_breath = false;
 };
 
+// Transition from PEEP to PIP pressure over this length of time.  Citation:
+// https://respiraworks.slack.com/archives/C011CJQV4Q7/p1591763842312500?thread_ts=1591759016.310200&cid=C011CJQV4Q7
+static constexpr Duration RISE_TIME = milliseconds(100);
```
Comment (jlebar) [inline controller/lib/core/blower_fsm.h:89]: `static` global functions or global variables should only appear in cc files.  In header files, you want to mark global functions or global variables as `inline`.

`static` means "private to this translation unit`, so a static variable/function in a header gets duplicated into every cpp that includes it!

(To maximize confusion, `static` inside of a class means something entirely different.  So this variable, previously inside of a class, was correctly marked as `static inline`.  Whereas variables outside of a class should never be `static inline`.)

File: controller/test/blower_fsm/blower_fsm_test.cpp [hunk 2]
```diff
@@ -69,6 +69,13 @@ void testSequence(
   }
 }
 
+constexpr int64_t rise_time_us = RISE_TIME.microseconds();
+static_assert(rise_time_us % 1000 == 0,
+              "blower fsm tests assume rise time is a whole number of ms.");
+constexpr int64_t rise_time_ms = rise_time_us / 1000;
+static_assert(rise_time_ms % 4 == 0,
+              "blower fsm tests assume we can divide rise time ms by 4.");
```
Comment (jlebar) [inline controller/test/blower_fsm/blower_fsm_test.cpp:77]: Put these global variables at the top alongside inputs_zero?

File: controller/lib/core/blower_fsm.h (hunk 1 above)
Comment (asmodai27) [reply controller/lib/core/blower_fsm.h:89]: Woops, hasty cut/paste on my part. Thanks for the clarification as to why it was static inline to begin with.

File: controller/test/blower_fsm/blower_fsm_test.cpp (hunk 2 above)
Comment (asmodai27) [reply controller/test/blower_fsm/blower_fsm_test.cpp:77]: sure thing.

**Pull request merged**